src/ansys/fluent/core/workflow.py

```diff
@@ -380,17 +380,20 @@ def python_name(self) -> str:
         return self._python_name
 
     def _set_python_name(self, compound=False):
-        this_command = self._command()
-        if compound:
-            p_name = (
-                self._command_source._compound_parent_task_python_name_id[0]
-                + f"_child_{self._command_source._compound_parent_task_python_name_id[1]}"
-            )
-            self._python_name = p_name
-            self._command_source._compound_task_map[self.name()] = p_name
-        else:
-            self._python_name = camel_to_snake_case(this_command.get_attr("helpString"))
-        self._cache_data(this_command)
+        if self._dynamic_interface:
+            this_command = self._command()
+            if compound:
+                p_name = (
+                    self._command_source._compound_parent_task_python_name_id[0]
+                    + f"_child_{self._command_source._compound_parent_task_python_name_id[1]}"
+                )
+                self._python_name = p_name
+                self._command_source._compound_task_map[self.name()] = p_name
+            else:
+                self._python_name = camel_to_snake_case(
+                    this_command.get_attr("helpString")
+                )
+            self._cache_data(this_command)
```

Comment:
@prmukherj:

Subsequently, we shall eliminate this code also:

![image](https://github.com/ansys/pyfluent/assets/93727996/6fe3ecc0-90bf-4d68-9fc4-1f7dd53cdd5c)

because it is a bug.

Reply:
ok, sure. I'll investigate into this.